tests/runner/iframeTestHarness.js

```diff
@@ -0,0 +1,20 @@
+define('iframeTestHarness', [
+    '$'
+], function(_$) {
+
+    var setUp = function(suite, done) {
+        var $frame = _$('<iframe />').attr('src', '/tests/fixtures/iframe_pinny.html');
+        _$('#frameContainer').html($frame);
+
+        _$(window).one('message', function() {
+            var frameWindow = $frame[0].contentWindow;
+
+            suite(frameWindow.$, frameWindow.modalCenter, $frame);
```

Comment:
So I'm feeling now that we should be appending our dependencies to a global dependencies object. This will allow us to do:

```
suite(frameWindow.$, frameWindow.dependencies, $frame);
```